include/cxx/cunistd

```diff
@@ -71,7 +71,9 @@ namespace std
 
   // Terminal I/O
 
+#ifdef CONFIG_SERIAL_TERMIOS
```

Comment:
let's move this change to patch "include/termios.h: undefine symbols when !CONFIG_SERIAL_TERMIOS"

Reply:
done

Reply:
I still see the change there

Reply:
The change is required, @GUIDINGLI already merge the change related CONFIG_SERIAL_TERMIOS into one patch.

Reply:
Ah, I thought I wanted to have this in a separate PR. Ok, I'll merge then